docs: Mention EOL for version 2 (#2238)
SDA-SE/sda-dropwizard-commons: README.md

>
> New features can only be contributed in version 3.x.x.
> 

> > Version 2.x.x only receives critical security updates until October 2023. We can only provide
> > security updates for version 2 if libraries are still compatible with Java 8.

Reviewer: ```suggestion
> Version 2.x.x only receives critical security updates until October 2023. We will only provide
> security updates for version 2 for the libraries that are still compatible with Java 8.
```

> > Version 2.x.x only receives critical security updates until October 2023. We can only provide
> > security updates for version 2 if libraries are still compatible with Java 8.

Reviewer 2: Nitpicking formatting:
```suggestion
> Version 2.x.x only receives critical security updates until October 2023.
> We can only provide security updates for version 2 if libraries are still compatible with Java 8.
```

Including the suggestion by @preethivenkatesh-sdase:
```suggestion
> Version 2.x.x only receives critical security updates until October 2023.
> We will only provide security updates for version 2 for the libraries that are still compatible with Java 8.
```
